source/leuven/genius_quick_start.rst

```diff
@@ -94,8 +104,24 @@ To specifically request V100 GPUs (which are on nodes with CascadeLake architect
   
 For the V100 type of GPU, it is required that you request 4 cores for each GPU. Also notice that these nodes offer much larger memory bank.
 
-Advanced usage
-^^^^^^^^^^^^^^
+Advanced node usage
```

Comment:
It's a bit weird to see this "Advanced usage" (sub?)section in here,
I would have expected it to see it in the "Submit to a compute node"
section (after the 'basic part' in that section). Wouldn't it be better
like that?

I.e.

- Submit to a compute node
  (basic info)
  - Advanced usage
    (advanced info)
- Submit to a GPU node
  (basic info)
  - Advanced usage
    (advanced info)

Reply:
The last commit should have taken care of all that.